app/editor/src/features/admin/products/ProductSubRequests.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { useProducts } from 'store/hooks/admin';
+import { IProductModel, IUserProductModel, Show } from 'tno-core';
+
+import * as styled from './styled';
+import { UserApproveDeny } from './UserApproveDeny';
+
+export const ProductSubRequests: React.FC = () => {
+  const { id } = useParams();
+  const [, { getProduct }] = useProducts();
+
+  const [product, setProduct] = React.useState<IProductModel>();
+  // users that are flagged to have a subscription request change reviewed
+  const [flaggedUsers, setFlaggedUsers] = React.useState<IUserProductModel[]>([]);
+
+  React.useEffect(() => {
+    if (Number(id) && !product) {
+      getProduct(Number(id)).then((result) => setProduct(result));
+    }
+  }, [id, product, getProduct]);
+
+  React.useEffect(() => {
+    if (!flaggedUsers.length && !!product) {
+      const flagged = product.subscribers.filter((user) => user.requestedIsSubscribedStatus);
```

Comment:
Wondering if future work will include `user_report` and `user_notification` to check subscriptions related.